I2S Speed test fails to build for LPCXpresso55S69

**Describe the bug**
`tests/drivers/i2s/i2s_speed` fails to build for the LPCXpresso55S69 platform, since c8dfdda02759b. The root cause of the issue is that not all SOCs support the secondary channel feature of the Flexcomm I2S, so the `I2S_EnableSecondaryChannel` function is not defined for all SOCs.

**To Reproduce**
Steps to reproduce the behavior:
1. west build -p always -b lpcxpresso55s69_cpu0 tests/drivers/i2s/i2s_speed
2. See error

**Expected behavior**
The build should pass

**Logs and console output**
```
/opt/zephyr-sdk-0.16.4/arm-zephyr-eabi/bin/../lib/gcc/arm-zephyr-eabi/12.2.0/../../../../arm-zephyr-eabi/bin/ld.bfd: zephyr/drivers/i2s/libdrivers__i2s.a(i2s_mcux_flexcomm.c.obj): in function `i2s_mcux_configure':
/home/danieldegrasse/zephyr/zephyrproject/zephyr/drivers/i2s/i2s_mcux_flexcomm.c:272: undefined reference to `I2S_EnableSecondaryChannel'
collect2: error: ld returned 1 exit status
ninja: build stopped: subcommand failed.
````

--- a/drivers/i2s/i2s_mcux_flexcomm.c
+++ b/drivers/i2s/i2s_mcux_flexcomm.c
@@ -264,6 +264,8 @@ static int i2s_mcux_configure(const struct device *dev, enum i2s_dir dir,
 		 * More than 2 channels are enabled, so we need to enable
 		 * secondary channel pairs.
 		 */
+#if (defined(FSL_FEATURE_I2S_SUPPORT_SECONDARY_CHANNEL) && \
+	FSL_FEATURE_I2S_SUPPORT_SECONDARY_CHANNEL)
 		for (uint32_t slot = 1; slot < i2s_cfg->channels / 2; slot++) {
 			/* Position must be set so that data does not overlap
 			 * with previous channel pair. Each channel pair
@@ -272,6 +274,10 @@ static int i2s_mcux_configure(const struct device *dev, enum i2s_dir dir,
 			I2S_EnableSecondaryChannel(cfg->base, slot - 1, false,
 						   i2s_cfg->word_size * 2 * slot);
 		}
+#else
+		/* No support */
+		return -ENOTSUP;
+#endif
 	}
 
 	/*

